app/src/app/api/v0/inventory/[inventory]/download/route.ts

```diff
@@ -223,12 +223,12 @@ async function inventoryXLS(inventory: Inventory): Promise<Buffer> {
       row.commit();
     }
   }
-  console.timeEnd("edit_ciris");
+  logger.debug({ timeEnd: Date.now() }, "edit_ciris");
 
   // await workbook.xlsx.writeFile("test.xlsx");
-  console.time("save_ciris");
+  logger.debug({ time: Date.now() }, "save_ciris");
   const buffer = Buffer.from(await workbook.xlsx.writeBuffer());
-  console.timeEnd("save_ciris");
+  logger.debug({ timeEnd: Date.now() }, "save_ciris");
```

Comment:
Might be better to log the duration rather than the two timestamps? (which is the behaviour of the console.time function that was used here originally).

Reply:
Same for the other instance of this in this file